pkg/gen/python-templates/kind.py.mustache

```diff
@@ -9,7 +9,7 @@ from ... import tables
 
 class {{Kind}}(pulumi.CustomResource):
     {{{Comment}}}
-    def __init__(self, __name__, __opts__=None{{#Properties}}, {{LanguageName}}=None{{/Properties}}):
+    def __init__(self, __name__, __opts__=None{{#Properties}}, {{LanguageName}}=None{{/Properties}}, **kwargs):
```

Comment:
Why do we need this change?

Reply:
So the problem here was that the Python SDK expects kwargs for the k8s resource fields rather than an object containing all of the fields. In the previous YAML code, I wasn't generating correct calls to the resource classes because of this difference. 

This approach just splats the `obj` and allows the classes to use what they need. I can't remember what field(s) it was, but `obj` had some kwargs that were not expected by the classes. That's perhaps a bit of a blunt/naive approach, but it's easy and works.

An alternative approach would be updating `yaml.py` to strip out any unneeded kwargs from `obj` prior to splatting it in the class constructors. This shouldn't require the extra `**kwargs` catch-all in the resource classes, but might be a little tricker to figure out. I can see how the first approach might be confusing to users though, so maybe worth the effort?

What do you think?

Reply:
I used the second approach and merged that change in #545